### DISCUSSION TITLE
[Question] Objection to the use of StaleBot

### DISCUSSION THREAD

**Discussion by nhojpatrick** [crc-org/crc#3430] [General]
### Steps to reproduce

  1. Raise Bug
  2. Waiting a maintainer to deal with issue
  3. StaleBot marks it as stale as no maintainer has dealt with issue
  4. StaleBot closes issue as no maintainer has dealt with issue

### Expected
StaleBot can't mark an issue as stale until a maintainer as reviewed and commented, it's fine to mark it as stale is the issue author doesn't get back to you. 

### Actual
Issues get closed within being addressed

**Comment by gbraad** [comment]
If people attend to the bugs, like the maintainers, the use of a stalebot is not necessarily wrong. Many of the issues that get closed are those to which the reporter did not follow up on a request. I understand the frustration if an issue gets closed after a while, but both sides are responsible for keeping the conversation happening. Our team is disciplined to review the issues that have been closed and pin them accordingly or follow up if we were to blame. If this didn't happen to a bug of you, please let us know.

-----

To clarify; this user hasn't filed an issue on our repository: https://github.com/crc-org/crc/issues?q=is%3Aissue+nhojpatrick
